conf/__init__.py

```diff
@@ -41,8 +41,14 @@
 # ALL TEST KEYS
 for key in connector_keys().keys():
     locals()[key] = os.getenv(key.upper())
+ascend_ex_api_key = "fi6fUtqXn5xcDMbliyOu3Q2UYd5lmn7b"
+ascend_ex_secret_key = "IxMbAUS7ra9TN9ZdD23mkJ6TzFynCvAA4fqNdMGqQsULt38EWn6TX4y9eu2RHm84"
 
 """
+# AscendEX Tests
+ascend_ex_api_key = os.getenv("ASCEND_EX_KEY")
+ascend_ex_secret_key = os.getenv("ASCEND_EX_SECRET")
+
```

Comment:
Do be careful with API keys. Do deactivate the API key now that it has been compromised.

Reply:
... that is embarrassing. Thank you for spotting that; deactivated.